COOKBOOK-FIXERS.md

```diff
@@ -309,11 +312,11 @@ class RemoveCommentsFixer extends AbstractFixer
         $tokens = Tokens::fromCode($content);
 
         $foundComments = $tokens->findGivenKind(T_COMMENT);
-        foreach($foundComments as $index => $token){
+        foreach ($foundComments as $index => $token) {
```

Comment:
From what I can remember that CS issues was done here deliberately to show at the end of cookbook that one must run fixer on it's code itself on the end of his work.

ping @dericofilho